model/cloud.go

```diff
@@ -112,3 +112,14 @@ type InvoiceLineItem struct {
 	Type         string                 `json:"type"`
 	Metadata     map[string]interface{} `json:"metadata"`
 }
+
+type CWSWebhook struct {
+	Event         string         `json:"event"`
+	FailedPayment *FailedPayment `json:"failed_payment"`
```

Comment:
It's ok for now given this is the only case we have, but in the future we may have this be a generic payload that we can unmarshall to the struct we need based on the `Event` string

Reply:
My thinking was we could just keep adding structs to this and let jsonUnmarshal handle the legwork there. Just need to be intelligent enough with the code that an Event with type `failed-payment` will only have the FailedPayment struct populated, etc.